src/pages/index.astro

```diff
@@ -66,10 +66,18 @@ import Sponsor from "../components/Sponsor.astro";
   <Section extraClasses="pb-4">
     <SectionHeading>Tickets</SectionHeading>
     <SectionParagraph>
-      Ticket sales will start immediately at <a
-        href="https://tickets.nixcon.org/2023/"
-        rel="external">https://tickets.nixcon.org/2023/</a
-      >. If you are unable to afford a ticket, please contact us at
+      Tickets are all taken!<br />
+      Live streaming will be available. Follow any of the NixOS social accounts for updates.
```

Comment:
Can we list the social accounts or point to where they are listed?

Sorry for nitpicking 😀

Reply:
Yes of course, no worries.

It could link to https://nixos.org/community/, or list out the same?

Reply:
They aren't the same. Would  be good to get a list of those from the Marketing team.

Reply:
Maybe we could add a social section somewhere in the site, maybe at the header or footer where most users would expect it.

Reply:
Can we include:
- https://chaos.social/@nixos_org
- https://twitter.com/nixos_org

Since this is where we are currently posting the updates.

Reply:
Let's ship it